test/pageobjects/PublicationCreatePage.js

```diff
@@ -0,0 +1,38 @@
+const Page = require('./Page');
+
+class PublicationCreatePage extends Page {
+
+    get titleInputField(){
+        return $("//input[@id = 'title']");
```

Comment:
You can use CSS selector instead of XPath which is faster in this case and you have a higher chance of finding your elements if something changed in the DOM
```suggestion
        return $('#title');
```

Reply:
Thanks for your review! I was using Xpath because they have already been used on other pages.  I deleted unnecessary files from my pull request.

> You can use CSS selector instead of XPath which is faster in this case and you have a higher chance of finding your elements if something changed in the DOM

Thanks for your review! I was using Xpath because they have already been used on other pages.  I deleted unnecessary files from my pull request.